docs/combine.rst

```diff
@@ -0,0 +1,105 @@
+combine
+=======
+
+.. dfhack-tool::
+    :summary: Combine stacks of food and plant from one or every stockpile.
+    :tags: fort productivity stockpiles items plants
+
+
+Usage
+-----
+
+::
+
+    combine [help] [<mode opts>] [<stockpile opts>] [<stack opts>] [<output opts>]
+
+
+Examples
+--------
+``combine``
+    Displays help
+``combine --preview --stockpile=all``
```

Comment:
a semi-standard that we're moving towards is to name options like this `-d`, `--dry-run`

Reply:
Yes, I can use dry run.

Reply:
done.